site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -1,158 +1,163 @@
+ port, DNS domain, and TLS certificate.
+- The store team has a single Service called _store_ that they have deployed
+in the `store-ns` Namespace which also needs to be exposed behind the same IP
+ address and domain.
+- The Foobar Corporation operates behind the `foo.example.com` domain for all
+ apps. This is controlled by a central infrastructure team, operating in the
+  `infra-ns` Namespace.
+- Lastly, the security team controls the certificate for `foo.example.com`.
+By managing this certificate through the single shared Gateway they are able
 to centrally control security without directly involving application teams.
 
 The logical relationship between the Gateway API resources looks like this:
 
 ![Cross-Namespace routing](../images/cross-namespace-routing.svg)
 
-## Cross-namespace Route binding
+## Cross-namespace Route Attachment
 
-[Route binding](/concepts/api-overview/#route-binding) is an important concept 
-that dictates how Routes and Gateways select each other to apply routing
-configuration to a Gateway. It is especially relevant when there are multiple
-Gateways and multiple Namespaces in a cluster. Gateway and Route binding is
-bidirectional - a binding can only exist if the Gateway owner and Route owner
-owner both agree to the relationship. This bi-directional relationship exists
-for two reasons:
+[Route attachment][attaching] is an important concept that dictates how Routes
+and Gateways select each other to apply routing configuration to a Gateway. It
+is especially relevant when there are multiple Gateways and multiple Namespaces
+in a cluster. Gateway and Route attachment is bidirectional - attachment can
+only succeed if the Gateway owner and Route owner owner both agree to the
+relationship. This bi-directional relationship exists for two reasons:
 
-- Route owners don't want to overexpose their applications and don't want 
+- Route owners don't want to overexpose their applications and don't want
 their apps to be accessible through paths they are not aware of.
-- Gateway owners don't want apps using certain Gateways they should not be 
-using. An internal application shouldn't be exposed through a public Gateway 
+- Gateway owners don't want apps using certain Gateways they should not be
+using. An internal application shouldn't be exposed through a public Gateway
 for example.
 
+As a result, Gateways and Routes support independent _Attachment Constraints_.
+Attachment constraints are selectors or references on a given resource which 
+determine the resources that may attach to it. Gateway and Route attachment 
+constraints create a handshake between the infra owners and the
+ application owners that enables them to be independent actors. Routes
+can only attach to specified Gateways and Gateways can constrain the Namespaces
+and type of Routes which are able to attach to it. Clusters which use attachment
+constraints to govern Route and Gateway attachment require less central administration
```

Comment:
I found this section a bit confusing, but I'm not sure my suggestion here is better, feel free to ignore.
```suggestion
application owners that enables them to be independent actors. App owners directly reference 
the Gateway(s) they want their Routes to be attached to. Infra owners can constrain the
namespaces and types of Routes that can be attached to their Gateway(s). Clusters which use
these attachment constraints to govern Route and Gateway attachment require less central 
administration
```